src/ir/control.rs

```diff
@@ -97,6 +99,10 @@ pub struct Invoke {
     pub inst: InstIdx,
     // The ports defined by this invocation
     pub ports: Vec<PortIdx>,
+    // The events defined by this invocation
+    pub events: Vec<EventIdx>,
```

Comment:
This seems incorrect: Instances don't define events; invocations do. What is this supposed to represent?